### PR TITLE
Replace exception message headers

### PR DESCRIPTION
**Pull request by Kair0s3**

### CODE REVIEW

File: src/main/java/decodex/parser/Parser.java [hunk 1]
```diff
@@ -57,27 +57,29 @@ public String getCommandType(String userInput) throws ParserException {
...
+        String strippedUserInput = userInput.stripLeading();
+        String[] tokens = strippedUserInput.split(" ", -1);
 
-        if (argumentTokens.length > 1) {
-            String joinedArguments = String.join(" ", argumentTokens);
-            return joinedArguments;
+        String argumentString = String.join(" ",
+                Arrays.copyOfRange(tokens, STARTING_ARGUMENTS_INDEX, tokens.length));
+        if (argumentString.isEmpty()) {
+            throw new ParserException("[-] Your input is empty, please do \"input i am a string\"");
```
Comment (rizemon) [inline src/main/java/decodex/parser/Parser.java:80]: As this method is generally used for those that have an input string, perhaps change the message to something more general like `[-] Input is empty`?

File: src/main/java/decodex/parser/Parser.java [hunk 2]
```diff
@@ -57,27 +57,29 @@ public String getCommandType(String userInput) throws ParserException {
...
+                .filter(value -> !value.isBlank())
+                .toArray(size -> new String[size]);
+        if (argumentTokens.length == 0) {
+            throw new ParserException("[-] Module name is missing");
         }
-        String[] argumentTokens = Arrays.copyOfRange(tokens, STARTING_ARGUMENTS_INDEX, tokens.length);
+        return argumentTokens[MODULE_NAME_INDEX_IN_TOKENS];
+    }
+
+    public String getInputString(String userInput) throws ParserException {
```
Comment (rizemon) [inline src/main/java/decodex/parser/Parser.java:73]: Perhaps add some JavaDoc comments?

File: src/main/java/decodex/parser/Parser.java [hunk 3]
```diff
@@ -57,27 +57,29 @@ public String getCommandType(String userInput) throws ParserException {
...
     }
 
-    /**
-     * Returns the Argument portion of the user input as a whole string.
-     *
-     * @param userInput The input specified by the user.
-     * @return The argument portion of the user input as an array.
-     */
-    public String getUserArgument(String userInput) throws ParserException {
+    public String getModuleName(String userInput) throws ParserException {
```
Comment (rizemon) [inline src/main/java/decodex/parser/Parser.java:60]: Perhaps add some JavaDoc comments?

Comment (arraysius) [reply src/main/java/decodex/parser/Parser.java:60]: JavaDocs can be done in a separate issue/PR as it is not the intended change in this PR

File: src/main/java/decodex/parser/Parser.java (hunk 2 above)
Comment (arraysius) [reply src/main/java/decodex/parser/Parser.java:73]: See my other comment.

File: src/main/java/decodex/parser/Parser.java (hunk 1 above)
Comment (arraysius) [reply src/main/java/decodex/parser/Parser.java:80]: Agree

File: src/main/java/decodex/parser/Parser.java (hunk 3 above)
Comment (rizemon) [reply src/main/java/decodex/parser/Parser.java:60]: Even though it was not the intended change, code that were introduced should still be of a certain standard (e.g comments, code quality) before being merged in.

File: src/main/java/decodex/parser/Parser.java (hunk 2 above)
Comment (rizemon) [reply src/main/java/decodex/parser/Parser.java:73]: See my other comment.

**Review by arraysius — APPROVED** [review]
LGTM

**Pull request merged**